docs/design/coreclr/jit/viewing-jit-dumps.md

```diff
@@ -58,6 +41,78 @@ The first thing to do is setup the .NET Core app we want to dump. Here are the s
     }
     ```
 
+* Set the configuration variables you need (see below) and run your app. For example:
+
+    ```
+    C:\test>set DOTNET_JitDisasm=InefficientJoin
+    C:\test>dotnet run -c Release
```

Comment:
Not sure if it's worth noting here, but `dotnet run` will run msbuild, nuget, roslyn etc so when you define `JitDisam` any of that might pick it up (especially if you use *). Generally, we do:
```
dotnet build (or publish)
JitDisasm=Foo
<run final binary>
```